.registryrc

```diff
@@ -1 +1 @@
-3b9f67add3338f151154bc5db76bf4e296f018dd
+f8add13fa300d7e420082b7a449b7dc59a08144b
```

Comment:
update once https://github.com/hyperlane-xyz/hyperlane-registry/pull/278 is merged